modules/administration-guide/pages/limiting-the-number-of-workspaces-that-a-user-can-keep.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: PROCEDURE
+:navtitle: Limiting the number of workspaces that a user can keep
+:description: Limiting the number of workspaces that a user can keep
+:keywords: administration guide, number, workspaces
+:page-aliases: configuring-the-number-of-workspaces-that-a-user-can-create.adoc
+
+[id="limiting-the-number-of-workspaces-that-a-user-can-keep_{context}"]
+= Limiting the number of workspaces that a user can keep 
+
+{prod-short} enables users to keep several workspaces in their dashboards.
+
+By default, users can keep an unlimited number of workspaces.
+
+You can set a limit on the number of workspaces that a user can keep in the dashboard.
+
+.Procedure
+
+. Obtain the name of the {prod-short} namespace. The default is `{prod-namespace}`.
++
+[source,terminal,subs="+quotes"]
+----
+$ oc get checluster --all-namespaces \
+  -o=jsonpath="{.items[*].metadata.namespace}"
+----
+
+. Configure the `CheCluster` Custom Resource. See xref:using-the-cli-to-configure-the-checluster-custom-resource.adoc[].
++
+[source,yaml,subs="+quotes"]
+----
+spec:
+  components:
+    cheServer:
+      extraProperties:
+        CHE_LIMITS_USER_WORKSPACES_COUNT: "__<max_number_of_kept_workspaces_per_user>__"
+----
++
+[NOTE]
+====
+The default value of __<max_number_of_kept_workspaces_per_user>__ is `-1`, which means that users can keep an unlimited number of workspaces. If you set the value to a positive integer, users will be able to keep as many workspaces as the integer value.
+====
+
+. To apply the configuration from step 2, run the following command for the namespace that you obtained in step 1:
++
+[source,terminal,subs="+quotes,attributes"]
+----
+$ oc patch checluster/devspaces -n  _<{prod-namespace}>_ \ <1>
+--type='merge' -p '{"spec":{"components":{"cheServer":{"extraProperties":{"CHE_LIMITS_USER_WORKSPACES_COUNT":"__<max_number_of_kept_workspaces_per_user>__"}}}}}' <2>
+----
+<1> The {prod-short} namespace that you obtained in step 1.
+<2> The `__<max_number_of_kept_workspaces_per_user>__` that you set for `CHE_LIMITS_USER_WORKSPACES_COUNT` in step 2.
```

Comment:
```suggestion
<1> The namespace you got in step 1.
<2> The `__<max_number_of_workspaces_per_user>__` value you set in step 2.
```

Reply:
It looks like you're giving users the choice of doing this task as a using a declarative or imperative approach. Consider handling it this way: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#declarative-config-examples

Reply:
Thank you. I'll keep it in mind. In this case, `oc` is another step in the procedure, not an alternative.